sw/device/tests/penetrationtests/firmware/BUILD

```diff
@@ -0,0 +1,117 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "//rules/opentitan:defs.bzl",
+    "opentitan_binary",
+    "opentitan_test",
+    "silicon_params",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+FIRMWARE_DEPS_FPGA = [
+    "//sw/device/tests/penetrationtests/firmware/fi:ibex_fi",
+    "//sw/device/tests/penetrationtests/firmware/fi:otbn_fi",
+    "//sw/device/tests/penetrationtests/firmware/sca:aes_sca",
+    "//sw/device/tests/penetrationtests/firmware/sca:ibex_sca",
+    "//sw/device/tests/penetrationtests/firmware/sca:kmac_sca",
+    "//sw/device/tests/penetrationtests/firmware/sca:prng_sca",
+    "//sw/device/tests/penetrationtests/firmware/sca:sha3_sca",
+    "//sw/device/tests/penetrationtests/firmware/sca:trigger_sca",
+    "//sw/device/tests/penetrationtests/firmware/lib:extclk_sca_fi",
+    "//sw/device/lib/base:csr",
+    "//sw/device/lib/base:status",
+    "//sw/device/lib/crypto/drivers:entropy",
+    "//sw/device/lib/testing/test_framework:check",
+    "//sw/device/lib/testing/test_framework:ottf_main",
+    "//sw/device/lib/testing/test_framework:ujson_ottf",
+    "//sw/device/lib/ujson",
+
+    # Include all JSON commands.
+    "//sw/device/tests/penetrationtests/json:commands",
+]
+
+FIRMWARE_DEPS_FI = [
+    "//sw/device/tests/penetrationtests/firmware/fi:ibex_fi",
+    "//sw/device/tests/penetrationtests/firmware/fi:otbn_fi",
+    "//sw/device/tests/penetrationtests/firmware/lib:extclk_sca_fi",
+    "//sw/device/lib/base:csr",
+    "//sw/device/lib/base:status",
+    "//sw/device/lib/crypto/drivers:entropy",
+    "//sw/device/lib/testing/test_framework:check",
+    "//sw/device/lib/testing/test_framework:ottf_main",
+    "//sw/device/lib/testing/test_framework:ujson_ottf",
+    "//sw/device/lib/ujson",
+
+    # Include all JSON commands.
+    "//sw/device/tests/penetrationtests/json:commands",
+]
+
+FIRMWARE_DEPS_SCA = [
+    "//sw/device/tests/penetrationtests/firmware/sca:aes_sca",
+    "//sw/device/tests/penetrationtests/firmware/sca:ibex_sca",
+    "//sw/device/tests/penetrationtests/firmware/sca:kmac_sca",
+    "//sw/device/tests/penetrationtests/firmware/sca:prng_sca",
+    "//sw/device/tests/penetrationtests/firmware/sca:sha3_sca",
+    "//sw/device/tests/penetrationtests/firmware/sca:trigger_sca",
+    "//sw/device/tests/penetrationtests/firmware/lib:extclk_sca_fi",
+    "//sw/device/lib/base:csr",
+    "//sw/device/lib/base:status",
```

Comment:
Is this still needed? I don't think so.

Reply:
`base:status` is needed as it contains the `TRY()` function I am using. Previously, I had a copy of the `TRY()` function in my own version of the `status` lib, which now is removed.

Reply:
Thanks for explaining, sounds good.